kuma/wiki/jinja2/wiki/includes/spam_error.html

```diff
@@ -14,3 +14,10 @@
     contact the site administrators</a>.
     {% endtrans %}
 </p>
+{% if perms.wiki.add_revisionakismetsubmission %}
```

Comment:
I added myself to all the spam admin groups and then tried to submit some spam, mostly by copying other stuff that has been marked as spam, that takes some time. It's possible that what I'm really triggering is Akismet's sense that I'm making too many edits too quickly. Either way, that usually works for me.

A faster way to test that this error has been caught would be to create the test user account with the username `viagra-test-123`. All submissions from that account should be marked as spam. I got the error submitting content with that account as well even though the message shouldn't display to them since it has to evaluate the file either way.